front/src/modules/object-record/record-table/components/RecordTable.tsx

```diff
@@ -90,9 +125,34 @@ export const RecordTable = ({
 }: RecordTableProps) => {
   const tableBodyRef = useRef<HTMLDivElement>(null);
 
-  const { resetTableRowSelection, setRowSelectedState } = useRecordTable({
+  const tableRowIds = useRecoilValue(tableRowIdsState);
```

Comment:
Here we could use `numberOfTableRowsState` instead.

Reply:
Done